comictaggerlib/pagelisteditor.py

```diff
@@ -369,11 +369,12 @@ def emit_front_cover_change(self) -> None:
             self.first_front_page = self.get_first_front_cover()
             self.firstFrontCoverChanged.emit(self.first_front_page)
 
-    def set_metadata_style(self, data_style: str) -> None:
+    def set_metadata_style(self, data_styles: list[str]) -> None:
         # depending on the current data style, certain fields are disabled
-        if data_style:
-            self.metadata_style = data_style
+        if data_styles:
+            styles = [metadata_styles[style] for style in data_styles]
+            enabled_widgets = [attr for style in styles for attr in style.supported_attributes]
```

Comment:
This needs to be put into a proper for loop. Yes it's faster but it takes me longer than 10 seconds to figure out what is happening on this line